packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -27,7 +28,7 @@ const AssetSummary = observer(() => {
     const cr_account = !is_eu_user && !no_CR_account;
 
     //dont show loader if user has no respective regional account
```

Comment:
```suggestion
    const is_loading = (isUserSwitchingOrLoggingIn && hasValidRegionalAccount) 
                              || !is_landing_company_loaded 
                              || !last_update;

    if (is_loading) {
```

I wonder if those and and ors could be improved too 🤔